bench/bench_zfec.py

```diff
@@ -77,26 +76,35 @@ def _encode_data_fec(N):
 
 def bench(k, m):
     SIZE = 10**6
-    MAXREPS = 64
+    MAXREPS = 1000
     # for f in [_encode_file_stringy_easyfec, _encode_file_stringy, _encode_file, _encode_file_not_really,]:
     # for f in [_encode_file,]:
     # for f in [_encode_file_not_really, _encode_file_not_really_and_hash, _encode_file, _encode_file_and_hash,]:
     # for f in [_encode_data_not_really, _encode_data_easyfec, _encode_data_fec,]:
-    print "measuring encoding of data with K=%d, M=%d, reporting results in nanoseconds per byte after encoding %d bytes %d times in a row..." % (k, m, SIZE, MAXREPS)
+    print("measuring encoding of data with K=%d, M=%d, encoding %d bytes %d times in a row..." % (k, m, SIZE, MAXREPS))
     # for f in [_encode_data_fec, _encode_data_not_really]:
     for f in [_encode_data_fec]:
         def _init_func(size):
             return _make_new_rand_data(size, k, m)
         for BSIZE in [SIZE]:
-            results = benchutil.rep_bench(f, n=BSIZE, initfunc=_init_func, MAXREPS=MAXREPS, MAXTIME=None, UNITS_PER_SECOND=1000000000)
-            print "and now represented in MB/s..."
-            print
-            best = results['best']
-            mean = results['mean']
-            worst = results['worst']
-            print "best:  % 4.3f MB/sec" % (10**3 / best)
-            print "mean:  % 4.3f MB/sec" % (10**3 / mean)
-            print "worst: % 4.3f MB/sec" % (10**3 / worst)
+            _init_func(BSIZE)
+            start = time()
+            for _ in range(MAXREPS):
+                f(BSIZE)
+            elapsed = (time() - start) / MAXREPS
+            print("Average MB/s:", (BSIZE / (1024 * 1024)) / elapsed)
```

Comment:
The reports were units of 10 ** 3 before but now they're 2 ** 20 which makes comparison tricky.  There are some ancient numbers in the README so maybe sticking with 2 ** 20 makes sense.

Reply:
Maybe I'll just update the README, so it's data from modern CPU?